Logger saved at ./logs_and_metrics/OTB100/siamrpn_mobilev2_l234_dwxcorr/211/4_net_G.pth/log_06:15:59.txt
Ran experiment with command: "python ttattack_targeted.py --tracker_name=siamrpn_mobilev2_l234_dwxcorr --dataset=OTB100 --case=211 --gpu=1 --model_iter=4_net_G.pth --trajcase=11 --targetcase=SE --attack_universal"
